packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m

```diff
@@ -67,22 +68,98 @@
     NSArray *startAtValues = startAt;
     query = [query queryStartingAtValues:startAtValues];
   }
+  id startAtDocument = parameters[@"startAtDocument"];
+  if (startAtDocument) {
+    NSArray *startAtValues = startAtDocument;
```

Comment:
I think we should use maps instead of lists to communicate information that can't be serialized with a MethodChannel codec. I'm switch this PR over to doing that.